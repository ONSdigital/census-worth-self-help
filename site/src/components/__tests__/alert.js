import React from "react"
import renderer from "react-test-renderer"

import Alert from "../alert"

describe("Alert", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Alert title="test_title" content="this is test content" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})