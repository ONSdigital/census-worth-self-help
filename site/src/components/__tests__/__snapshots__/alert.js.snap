// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "15ac0on",
      "next": undefined,
      "styles": "
        border-radius: 4px;
        border: solid 2px rgb(248, 196, 30);
        background-color: rgb(253, 239, 173);
        margin: 15px 15px 0px 15px;
        
    padding: 15px;
  
      ",
    }
  }
>
  <div
    className="Notification-heading-Style"
    css={
      Object {
        "map": undefined,
        "name": "kmpsy0",
        "next": undefined,
        "styles": "
          padding-top: 5px;
          padding-bottom: 10px;
        ",
      }
    }
  >
    test_title
  </div>
  <div
    className="Notification-body-Style"
    css={
      Object {
        "map": undefined,
        "name": "1a5rwwh",
        "next": undefined,
        "styles": "
          padding-top: 10px;
          padding-bottom: 5px;
        ",
      }
    }
    data-testid="alert-message"
  >
    this is test content
  </div>
</div>
`;
